### ISSUE TITLE
Left hand traffic conversion issue.

### ISSUE THREAD

**Issue by hcLime** [daohu527/imap#111]
I'm trying to convert openDRIVE to HD Apollo map format. Road network describes left hand traffic scenery.
Once imported to Apollo, it recognises it as right hand traffic- the vehicle is pointing in the wrong direction.

Does IMAP left hand drive traffic conversion?

**Comment by daohu527** [comment]
Yes, left-hand rules are not currently supported. In order to simplify the coding during development, only one rule was supported for the time being.

This requires some options. I will determine how to support left-hand rules soon and feedback.
